plugins/kubernetes-backend/src/service/KubernetesBuilder.test.ts

```diff
@@ -312,17 +337,22 @@ describe('KubernetesBuilder', () => {
       expect(response.body).toStrictEqual(requestBody);
     });
 
-    it('supports yaml content type', async () => {
+    it('supports yaml content type with permission set to allow', async () => {
       const requestBody = `---
-kind: Namespace
-apiVersion: v1
-metadata:
-  name: new-ns
-`;
+    kind: Namespace
+    apiVersion: v1
+    metadata:
+      name: new-ns
```

Comment:
can we revert the indentation here? Even though it's nicer to look at, I think this might technically make the body invalid YAML
```suggestion
kind: Namespace
apiVersion: v1
metadata:
  name: new-ns
```

Reply:
Yes, auto formatter must of touched it while i added a new test. Reverted to previous format.